MDNetworking/Wrappers/MDList.cs

```diff
@@ -575,8 +658,9 @@ public MDDisposableList<T> GetRawList()
         public void DoFullResynch(List<T> NewList)
         {
             CheckIfUnsafeMode();
-            RealList = new List<T>(NewList);
-            FullResynch = true;
+            // TODO: Fix this again
```

Comment:
What's the issue with this?

Reply:
Because the internal list now is a list containing keyvaluepair<T, IMDDataConverter> we can't just translate it straight back. I guess for unsafe mode they would just get access to this inner list with the structure it has after the change. So they would just have to know that they also get the converter.